src/components/album/AlbumsList.js

```diff
@@ -22,41 +24,47 @@ import {
 
 const AlbumsList = () => {
   const [albums, setAlbums] = useState([]); // for albums
-  const [limit, setLimit] = useState(1); // for pagination
+  const [limit, setLimit] = useState(12); // for pagination
```

Comment:
Since we never need to change the `limit` / page size, we can probably make this a static variable instead of a part of the component state (unless we forsee some use for it in the future in which case Im fine to leave it as is)

Reply:
Code fixed according to them recommendations